### ISSUE TITLE
Failed to run `config` for indent-blankline.nvim

### ISSUE THREAD

**Issue by vadirajks** [siduck76/NvChad#3092]
Hi Team, need help to fix this error.
getting below error for the new setup:

```

`Failed to run `config` for indent-blankline.nvim
                                                                                                                                                                                                 
.../share/nvim/lazy/indent-blankline.nvim/lua/ibl/utils.lua:22: type: expected function: 0x730b1e8eae50, got string (WHITESPACE)                                                                 
                                                                                                                                                                                                 
# stacktrace:                                                                                                                                                                                    
  - vim/shared.lua:0 _in_ **validate**                                                                                                                                                           
  - /indent-blankline.nvim/lua/ibl/utils.lua:22 _in_ **validate**                                                                                                                                
  - /indent-blankline.nvim/lua/ibl/hooks.lua:61 _in_ **register**                                                                                                                                
  - /NvChad/lua/nvchad/plugins/init.lua:42 _in_ **config**                                                                                                                                       
  - .config/nvim/init.lua:17                                                                                                                                                                     
Press ENTER or type command to continue  `
```
![image](https://github.com/user-attachments/assets/3d8490d7-7380-4740-9e53-5c5e1c37060f)

**Comment by HugoPhi** [comment]
me too, but someone said it can be solved when you use neovim v0.10 [#3091](https://github.com/NvChad/NvChad/issues/3091#issuecomment-2466786743)

**Comment by siduck** [comment]
use v0.10. i think blankline has different config for nightly
